packages/common/container-definitions/src/runtime.ts

```diff
@@ -119,7 +119,7 @@ export interface IContainerContext extends IDisposable {
     readonly storage: IDocumentStorageService;
     readonly connected: boolean;
     readonly baseSnapshot: ISnapshotTree | undefined;
-    readonly submitFn: (type: MessageType, contents: any, batch: boolean, appData?: any) => number;
+    readonly submitFn: (type: MessageType, contents: any, maxBatchSize: number | undefined, appData?: any) => number;
```

Comment:
This would be a breaking change and need some kind of staging, particularly since this affects the loader/container API boundary.  It's expected that old versions of the loader need to load new versions of the container runtime, which will break due to mismatch of the parameter type here.

Reply:
I actually would have expected this to fail build?  Not exactly sure how it passed?

Reply:
Reverted